src/validators/discussion-validator.js

```diff
@@ -0,0 +1,100 @@
+const  {check, param}  =  require ( 'express-validator' )
+const {StatusCodes} = require ('http-status-codes');
+const {validateResult} = require('../helpers/response-result');
+const {validationId} = require('../validators/account-validator');
+
+
+const validationDiscussionFilters  = [
+    param('filter')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isIn(['news', 'populars', ]).withMessage('El filtro es inválido'),
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
        .isIn(["news", 'populars', ]).withMessage('El filtro es inválido'),
```